src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Document/Document.ts

```diff
@@ -72,6 +73,7 @@ export interface IScope extends angular.IScope {
 export interface IFormScope extends IScope {
     showError;
     addParagraph() : void;
+    deleteParagraph(number) : void;
```

Comment:
Needs to be `deleteParagraph(index : number) : void;`. Otherwise `number` is interpreted as a name rather than a type.